tracer/test/snapshots/Security.AspNetCore5AsmInitializationSecurityEnabled.TestSecurityInitialization.verified.txt

```diff
@@ -1,4 +1,4 @@
-[
```

Comment:
It appears confusing that the `Security.AspnetCore5AsmToggle` snapshot moved to the `Security.AspNetCore5AsmInitialzation` snapshot, but they were actually identical earlier so git mixed them up

Reply:
This also explains the snapshot renaming from `Security.AspNetCore5AsmInitialization` to `Security.AspNetCore5AsmToggleConfigError`